### DISCUSSION TITLE
Solution not available for some parameter variant(s)

### DISCUSSION THREAD

**Discussion by mia-K71** [IRIS-Solutions-Team/IRIS-Toolbox#361] [Q&A]
Hi, 

I get the warning below when I try to solve my model. I managed to solve it with only a couple of sets of parameters (but these parameterizations are quite unrealistic and cannot be used for further simulations).

Is there a way to determine which equation is causing the problem? 
Thank you!

Warning: IrisToolbox Warning @ model:solve.
*** Solution not available for some parameter variant(s)
    No stable solution [Parameter Variant(s) #1]
    [Bkw variables, Unit roots, Stable roots]: Parameter Variant(s) #1[10 2 7]

**Comment by jaromir-benes** [comment]
In general, this is a problem of the model as a whole simultaneous system.
Best, J
